utilities/dxgridmaker.cpp

```diff
@@ -38,7 +39,7 @@
 void usage()
 {
     std::cerr << "Usage: dxgridmaker [OPTIONS]" << std::endl;
-    std::cerr << "Analysis point generator version " << VERSIONMAJOR << "." << VERSIONMINOR << "." << VERSIONPATCH << std::endl;
+   // std::cerr << "Analysis point generator version " << VERSIONMAJOR << "." << VERSIONMINOR << "." << VERSIONPATCH << std::endl;
```

Comment:
This line was causing a problem that I could not resolve, but I just tested it in my latest setup and it is no longer a problem.

Rick

From: Jason DeGraw [mailto:notifications@github.com]
Sent: Wednesday, March 30, 2016 4:36 PM
To: Architectural-Lighting-Simulation/STADIC <STADIC@noreply.github.com>
Cc: RMistrick <RMistrick@psu.edu>
Subject: Re: [Architectural-Lighting-Simulation/STADIC] Adding PSU Modifications (#114)


In utilities/dxgridmaker.cpp<https://github.com/Architectural-Lighting-Simulation/STADIC/pull/114#discussion_r57959093>:

> @@ -38,7 +39,7 @@

>  void usage()

>  {

>      std::cerr << "Usage: dxgridmaker [OPTIONS]" << std::endl;

> -    std::cerr << "Analysis point generator version " << VERSIONMAJOR << "." << VERSIONMINOR << "." << VERSIONPATCH << std::endl;

> +   // std::cerr << "Analysis point generator version " << VERSIONMAJOR << "." << VERSIONMINOR << "." << VERSIONPATCH << std::endl;

Why is this commented out?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/Architectural-Lighting-Simulation/STADIC/pull/114/files/e70721b4ae48110a4bda4e7f0971d9e6cb44ae51#r57959093>